src/js/components/List/List.js

```diff
@@ -18,10 +18,26 @@ const StyledList = styled.ul`
       props.tabIndex >= 0 &&
       focusStyle({ forceOutline: true, skipSvgChildren: true })}
   }
+  // during the interim state when a user is holding down a click,
+  // the individual list item has focus in the DOM until the click
+  // completes and focus is placed back on the list container.
+  // for visual consistency, we want to keep the focus indicator on the
+  // list container the whole time.
+  ${props =>
+    props.itemFocus &&
+    focusStyle({ forceOutline: true, skipSvgChildren: true })}}
 `;
 
 const StyledItem = styled(Box)`
   ${props => props.onClick && `cursor: pointer;`}
+  // during the interim state when a user is holding down a click,
+  // the individual list item has focus in the DOM until the click
+  // completes and focus is placed back on the list container.
+  // for visual consistency, we are showing focus on the list container
+  // as opposed to the item itself.
+  &:focus {
+    outline: none;
```

Comment:
There are other styles of focus possible, based on how the theme works. I think we should consider adding an `unfocusStyle()` that parallels `focusStyle()` to handle that.

Reply:
I've created an `unfocusStyle()` function next to `focusStyle()` -- I added a comment in the file noting that it is being used in List for a specific case and that the general recommendation would be to not integrate this into other components.